frontend/components/password-change/pug/password-change.pug

```diff
@@ -15,6 +15,16 @@
 							ng-model='vm.newPassword'
 							ng-required
 							ng-keypress="vm.passwordChange($event)")
+					md-input-container.md-block
+						label(ng-if="vm.registerMessage !== undefined") Confirm Password {{vm.registerMessage}}
```

Comment:
vm.registerMessage never seems to show anything?

Reply:
The "Set Password" button is only activated when both passwords are correct and they pass the strength validator. 

I have also resolved a css bug i came across, which squashed the password change div on lower mobile breakpoints.

![squashedpassworddiv](https://user-images.githubusercontent.com/6556019/44779934-a9eb5e80-ab78-11e8-842d-d256848a6fe4.png)
![squashedpassworddivmobile](https://user-images.githubusercontent.com/6556019/44779935-aa83f500-ab78-11e8-9e99-0bbcbb750715.png)

The images below show the final result of css bug.

![resolvedcss](https://user-images.githubusercontent.com/6556019/44780144-2aaa5a80-ab79-11e8-8689-d3877a792300.png)
![resolvedcss2](https://user-images.githubusercontent.com/6556019/44780145-2aaa5a80-ab79-11e8-924a-c47ebf9c0c0e.png)